docs/general/ad-filtering/create-own-filters.md

```diff
@@ -610,6 +614,35 @@ There is a big difference in how AdGuard determines the content type on differen
 - `||example.org^$script,stylesheet` — corresponds to all the scripts and styles from `example.org`.
 - `||example.org^$~image,~script,~stylesheet` — corresponds to all requests to `example.org` except for the images, scripts and styles.
 
+<!-- Please keep them sorted -->
+
+| Modifier \ Products | [CL apps][cl-apps] | [ExtChr][ext-chr] | [ExtFf][ext-ff] | [iOS app][ios-app] | [Safari][ext-saf] | [CB][and-cb] |
+| --- | :---: | :---: | :---: | :---: | :---: | :---: |
+| [$document](#document-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [$font](#font-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$image](#image-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$media](#media-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$object](#object-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$other](#other-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [$ping](#ping-modifier) | ✅ * | ✅ | ✅ | ❌ | ❌ | ✅ |
+| [$script](#script-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$stylesheet](#stylesheet-modifier) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [$subdocument](#subdocument-modifier) | ? | ✅ | ✅ | ✅ | ✅ | ❌ |
```

Comment:
```suggestion
| [$subdocument](#subdocument-modifier) | ✅ * | ✅ | ✅ | ✅ | ✅ | ❌ |
```
CoreLibs detects subdocument by a value of the `sec-fetch-dest` header if it's present. Otherwise, some main pages may be treated as subdocuments.